### ISSUE TITLE
Not clear in README that params do not only apply to url params only.

### ISSUE THREAD

**Issue by Zelfapp** [ohmybrew/Basic-Shopify-API#29]
```
$api->rest(string $type, string $path, array $params = null);
```

This is minor, but it wasn't as easy to understand as it could be that when the ```type``` is something other then GET that params will be converted to JSON as the payload. Documentation only refers to "the path", which in my opinion points to the query string params on a GET.

> params refers to an array of params you wish to pass to the path, examples: ['handle' => 'cool-coat']

My suggestion would be to make it a little more clear how the ```type``` and the ```params``` operate for GET, POST, PUT, DELETE, etc.

* ```params``` refers to an array of params you wish to pass to the path, examples ```['handle' => 'cool-coat']```. When the ```type``` will perform an update such as a PUT ```params``` will be converted to JSON as the payload.

**Comment by ohmybrew** [comment]
Currently all but GET will convert to JSON payload. Waiting on a response to another issue before I confirm this, but I'll def update.

**Comment by ohmybrew** [comment]
@Zelfapp Would you mind updating the readme to something you think would help? I'm unsure how to word it.
